### ISSUE TITLE
assertion and SIGABRT when building Boo.NAnt.Tasks with Mono 6

### ISSUE THREAD

**Issue by tpokorra** [boo-lang/boo#202]
```
Boo.NAnt.Tasks:

     [exec] /root/rpmbuild/BUILD/boo-alpha/default.build(308,4): Attribute 'useruntimeengine' for <exec ... /> is deprecated.  Use the managed attribute and Managed property instead.
     [exec] Boo Compiler version 0.9.7.0 (6.6.0.161 (tarball Wed Feb  5 07:34:41 UTC 2020))
     [exec] * Assertion at sre-encode.c:290, condition `count > 0' not met
     [exec] 
     [exec] 
     [exec] =================================================================
     [exec]  Native Crash Reporting
     [exec] =================================================================
     [exec] Got a SIGABRT while executing native code. This usually indicates
     [exec] a fatal error in the mono runtime or one of the native libraries 
     [exec] used by your application.
     [exec] =================================================================
```

for details, see the stacktrace at https://bugzilla.redhat.com/show_bug.cgi?id=1799197#c6

**Comment by tpokorra** [comment]
sorry for the noise.
it seems, this was already reported, and fixed in Mono 6.10:
https://xamarin.github.io/bugzilla-archives/45/45136/bug.html

**Comment by tpokorra** [comment]
actually, that bug was fixed in Mono 5.0.
I have filed a new bug report with Mono: https://github.com/mono/mono/issues/18970
